repository/index.js

```diff
@@ -170,20 +169,84 @@ const getAgendaURI = async (newAgendaId) => {
   return data.results.bindings[0].agenda.value;
 };
 
+/**
+ * Deletes agendaitems for a specific agenda
+ * @name deleteAgendaitems
+ * @function
+ * @param {String} deleteAgendaURI - The URI of the agenda to delete the agendaitems from
+ */
 const deleteAgendaitems = async (deleteAgendaURI) => {
+  const agendaItemUrisQueryResult = await selectAgendaItems(deleteAgendaURI);
+  const uriList = agendaItemUrisQueryResult.results.bindings;
+  const listOfAgendaItemUris = uriList.map((uri) => { return uri.agendaitem.value});
```

Comment:
Het verwerken van het resultaat van deze query zou beter gebeuren in de selectAgendaItems methode ?
in de util staat een methode "parseSparqlResults"
Kunnen we die niet gebruiken om het resultaat te verwerken, die methode gebruiken we in veel van onze services

Reply:
fixed